_clients/javascript/helpers.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: default
+title: Helper methods
+parent: JavaScript client
+nav_order: 2
+---
+
+# Helper methods
+
+This section is currently incomplete. It does not yet cover all helper methods and their options.
+{: .warning }
+
+Helper methods simplify the use of complicated API tasks.
+
+## Bulk helper
+
+Running bulk requests can be complex because of the API structure. The bulk helper aims to provide a nicer developer experience around the Bulk API.
```

Comment:
```suggestion
Running bulk requests can be complex because of the API structure. The bulk helper aims simplifies the developer experience with the Bulk API.
```